packages/devtools_app/lib/src/timeline/flutter/timeline_processor.dart

```diff
@@ -972,29 +473,125 @@ class FullTimelineProcessor extends TimelineProcessor {
     }
   }
 
-  @override
+  void _handleFrameStartEvent(TraceEvent event) {
+    if (event.id != null) {
+      final pendingFrame = _frameFromEvent(event);
+      pendingFrame.pipelineItemTime.start = Duration(
+        microseconds: nullSafeMin(
+          pendingFrame.pipelineItemTime.start?.inMicroseconds,
+          event.timestampMicros,
+        ),
+      );
+      if (pendingFrame.pipelineItemTime.start.inMicroseconds ==
+          event.timestampMicros) {
+        pendingFrame.pipelineItemStartTrace = event;
+      }
+      _maybeAddCompletedFrame(pendingFrame);
+    }
+  }
+
+  void _handleFrameEndEvent(TraceEvent event) async {
+    if (event.id != null) {
+      // Since we handle events in ascending timestamp order, we should ignore
+      // frame end events that we receive before the corresponding frame start
+      // event.
+      if (!pendingFrames.containsKey(_frameId(event))) return;
+      final pendingFrame = _frameFromEvent(event);
+      pendingFrame.pipelineItemTime.end = Duration(
+        microseconds: nullSafeMax(
+          pendingFrame.pipelineItemTime.end?.inMicroseconds,
+          event.timestampMicros,
+        ),
+      );
+      if (pendingFrame.pipelineItemTime.end.inMicroseconds ==
+          event.timestampMicros) {
+        pendingFrame.pipelineItemEndTrace = event;
+      }
+      _maybeAddCompletedFrame(pendingFrame);
+    }
+  }
+
+  /// Add event to an available frame in [pendingFrames] if we can.
+  void _maybeAddFlutterFrameEvent(SyncTimelineEvent event) {
+    if (!event.isUiEventFlow && !event.isGpuEventFlow) {
+      // We do not care about events that are neither the main flow of UI
```

Comment:
add a TODO and bug to figure out how to still show these events.

Reply:
we do still show these events, we just don't assign them to a Flutter frame.